### ISSUE TITLE
Chapter2 examples from book not compiling: "unknown module 'Math'"

### ISSUE THREAD

**Issue by jlipps** [purescript/purescript#626]
Here's the file I've constructed:

```
module Chapter2 where

import Math
import Debug.Trace

diagonal w h = sqrt (w * w + h * h)

main = print (diagonal 3 4)
```

When compiling using grunt or `psc src/Chapter2.purs --output dist/Main.js --main Chapter2 --module Chapter2`, I get this error:

```
Error at src/Chapter2.purs line 3, column 1:
Error in module Chapter2
Cannot import unknown module 'Math'
```

Where am I going wrong?

**Comment by ltfschoen** [comment]
@jlipps Jonathan, did you figure out what was causing the ```Cannot import unknown module 'Math'``` error?

**Comment by jlipps** [comment]
yes, I needed to install some stuff using Bower, as described in the book.

**Comment by ltfschoen** [comment]
@jlipps I have encountered the same error as you, but even after using Bower to install all the dependencies (as described in the book). Since this Issue is already closed, I created Issue [#797](https://github.com/purescript/purescript/issues/797). I have shared my code, which includes setup instructions and code commentary in my repo [here](https://github.com/ltfschoen/PureScriptByExample)

**Comment by jlipps** [comment]
Ah I see. Unfortunately I don't remember doing anything other than going
back to the instructions I skipped and then running them.
